beets/autotag/hooks.py

```diff
@@ -214,16 +235,23 @@ def __init__(
 
 
 # Structures that compose all the information for a candidate match.
+@dataclass
+class Match:
+    distance: Distance
+    info: Info
 
 
-class AlbumMatch(NamedTuple):
-    distance: Distance
+@dataclass
```

Comment:
Do we really need dataclasses here? They introduce a bit of overhead compared to named tuples. We are not doing any instance checks and if we remove the post init I dont see the added value.